database.sql

```diff
@@ -608,7 +608,7 @@ CREATE TABLE IF NOT EXISTS `diaspora-contact` (
 	`gsid` int unsigned COMMENT 'Global Server ID',
 	`created` datetime NOT NULL DEFAULT '0001-01-01 00:00:00' COMMENT '',
 	`updated` datetime NOT NULL DEFAULT '0001-01-01 00:00:00' COMMENT '',
-	`interacting_count` int unsigned DEFAULT 0 COMMENT 'Number of contacts this contact interactes with',
+	`interacting_count` int unsigned DEFAULT 0 COMMENT 'Number of contacts this contact interacts with',
```

Comment:
I think the previous round also touched things like this, so I think we decided it's ok...

Reply:
As long as you also update `static/dbstructure.config.php` and the relevant Markdown documentation page, it's ok!